tests/test_transforms/test_inference/test_inverse_transform.py

```diff
@@ -836,46 +856,13 @@ def _test_inverse_transform_future_new_segments(self, ts, transform, train_segme
         "transform, dataset_name, expected_changes",
         [
             # encoders
-            (LabelEncoderTransform(in_column="weekday", out_column="res"), "ts_with_exog", {}),
-            (
-                OneHotEncoderTransform(in_column="weekday", out_column="res"),
-                "ts_with_exog",
-                {},
-            ),
-            # feature_selection
-            (FilterFeaturesTransform(exclude=["year"]), "ts_with_exog", {}),
-            (
-                GaleShapleyFeatureSelectionTransform(relevance_table=StatisticsRelevanceTable(), top_k=2),
-                "ts_with_exog",
-                {},
-            ),
-            (
-                MRMRFeatureSelectionTransform(
-                    relevance_table=StatisticsRelevanceTable(), top_k=2, fast_redundancy=True
-                ),
-                "ts_with_exog",
-                {},
-            ),
```

Comment:
Why do you delete this transforms from tests?

Reply:
I did it by accident